src/helpers/dom/element.js

```diff
@@ -1083,15 +1021,7 @@ export function hasHorizontalScrollbar(element) {
  * Sets overlay position depending on it's type and used browser
  */
 export function setOverlayPosition(overlayElem, left, top) {
-  if (isIE8() || isIE9()) {
-    overlayElem.style.top = top;
-    overlayElem.style.left = left;
-  } else if (isSafari()) {
-    overlayElem.style['-webkit-transform'] = `translate3d(${left},${top},0)`;
-    overlayElem.style['-webkit-transform'] = `translate3d(${left},${top},0)`;
-  } else {
-    overlayElem.style.transform = `translate3d(${left},${top},0)`;
-  }
+  overlayElem.style.transform = `translate3d(${left},${top},0)`;
```

Comment:
It seems that `translate3d` won't work for the IE9, but we still support it.

https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function#Browser_compatibility
https://caniuse.com/#feat=transforms3d

Reply:
Yep, 👍  I went too far. I've reverted the [change](https://github.com/handsontable/handsontable/pull/6625/commits/b620dae739c195a5d9d456fa964e4303507219ff).